src/patternfly/components/Tabs/examples/Tabs.md

```diff
@@ -177,3 +186,4 @@ Whenever a list of tabs is unique on the current page, it can be used in a `<nav
 | `.pf-m-fill` | `.pf-c-tabs` | Modifies the tabs to fill the available space. |
 | `.pf-m-current` | `.pf-c-tabs__item` | Indicates that a tab item is currently selected. |
 | `.pf-m-inset-{none, sm, md, lg, xl, 2xl}{-on-[md, lg, xl, 2xl]}` | `.pf-c-tabs` | Modifies the tabs component padding/inset to visually match padding of other adjacent components. |
+| `. pf-m-alt-color-scheme` | `.pf-c-tabs` | Modifies the tabs component tab background colors. |
```

Comment:
@lboehling @mcarrano we're calling this variation `pf-m-alt-color-scheme`. Is that a good name or is there a more meaningful name we could use? Also thinking forward, would the name scale well if we introduced another color variation for use in a different context?

Reply:
Not sure @mcoker . The purpose of this is to just swap the white for gray, correct? If so, should be just call this something more like 'inverted'?